code/game/machinery/computer/robot.dm

```diff
@@ -3,11 +3,25 @@
 	desc = "Used to remotely lockdown or detonate linked Cyborgs and Drones."
 	icon_screen = "robot"
 	icon_keyboard = "rd_key"
-	req_access = list(ACCESS_ROBOTICS)
+	req_access = list(ACCESS_RD)
```

Comment:
wouldn't `ACCESS_AI_UPLOAD` be better? if you got access to AI upload you should be able to change laws

Reply:
It would make you need upload access to lock borgs for example, both roles that start with one start with the other anyways